enforcer_interface.go

```diff
@@ -30,12 +30,14 @@ type IEnforcer interface {
 	InitWithFile(modelPath string, policyPath string) error
 	InitWithAdapter(modelPath string, adapter persist.Adapter) error
 	InitWithModelAndAdapter(m model.Model, adapter persist.Adapter) error
+	initialize()
```

Comment:
We should not add the private method to the `interface`. 

WDYT?

Reply:
Actually it doesn't matter if we add it or not, but if we add it, ```initialize()``` becomes a compulsory implemented member function, as it is important in any of the ```Enforcer``` variants.